packages/orchestration/src/typeGuards.js

```diff
@@ -164,7 +163,10 @@ export const chainFacadeMethods = harden({
  * `seconds` is a big integer but since it goes through JSON it is encoded as
  * string
  */
-export const TimestampProtoShape = { seconds: M.string(), nanos: M.number() };
+export const TimestampProtoShape = harden({
+  seconds: M.string(),
+  nanos: M.number(),
+});
```

Comment:
consider hardening below the export. reads cleaner and doesn't affect the blame history:
```suggestion
export const TimestampProtoShape = { seconds: M.string(), nanos: M.number() };
harden(TimestampProtoShape)
```

It's also supported by [@endo/harden-exports](https://github.com/endojs/endo/blob/master/packages/eslint-plugin/lib/rules/harden-exports.js) to verify all exports are hardened.

Reply:
Cool that we have `@endo/harden-exports`! Why wasn't it warning about these cases?

Reply:
It's only enabled on `*.flows.js`. And I don't remember whether it detects `harden` in assignments

Reply:
Done, including to `AnyNatAmountShape` and `chainFacadeMethods`, which had been defined the other way, for consistency.

Note that outside this file there is still a tremendous number of declarations in the old inline harden style, which becomes a more distant inconsistency. Will be good to enable the `@endo/harden-exports` warning and fix all these!

Reply:
Done! I saw this PR wasn't merging and would need another round of CI anyway, I so made it convert all of them.